SlevomatCodingStandard/Sniffs/Classes/RequireSingleLineMethodSignatureSniff.php

```diff
@@ -36,10 +36,10 @@ public function process(File $phpcsFile, $methodPointer): void
 		}
 
 		$signature = $this->getSignature($phpcsFile, $signatureStartPointer, $signatureEndPointer);
```

Comment:
Please revert all changes in this file.

Reply:
Why? There is a typo + "double normalization", both should be addressed.

Reply:
Because it's not relevant to the bug. And your typo fix is not right :) "Indention" is wrong too :)

https://translate.google.cz/#view=home&op=translate&sl=en&tl=cs&text=indentation

Reply:
I just used the first one that Phpstorm offered. According to the dictionary it's correct, but archaic ... alright, so fix in spearate commit or you wan't completely separate pr?

Reply:
Just ignore it :) I will fix it by myself but I'm happy you've found it :)